src/App.php

```diff
@@ -144,13 +151,16 @@ public function registerFooterScript($path)
 	/**
 	 * @brief App constructor.
 	 *
-	 * @param string $basePath  Path to the app base folder
-	 * @param bool   $isBackend Whether it is used for backend or frontend (Default true=backend)
+	 * @param string           $basePath  Path to the app base folder
+	 * @param LoggerInterface  $logger    Logger of this application
+	 * @param bool             $isBackend Whether it is used for backend or frontend (Default true=backend)
 	 *
 	 * @throws Exception if the Basepath is not usable
 	 */
-	public function __construct($basePath, $isBackend = true)
+	public function __construct($basePath, $logger, $isBackend = true)
```

Comment:
You can type-hint `LoggerInterface` for the second parameter of this constructor.